packages/app/components/Settings/SettingsPage.tsx

```diff
@@ -66,7 +66,7 @@ export default function SettingsPage({
   useEffect(() => {
     const widthDivider = isMobile ? 6 : 10
     setAvatarSize(windowWidth / widthDivider)
-  })
+  }, [windowWidth, isMobile])
```

Comment:
I still hate this isMobile hook and I think we should burn it. It causes re-renders of the whole component whenever the resolution of the page changes. This is what CSS is for.